services/QuillLMS/app/controllers/api/v1/app_settings_controller.rb

```diff
@@ -0,0 +1,22 @@
+class Api::V1::AppSettingsController < ApplicationController
+  def index
+    user = User.find(app_setting_index_params)
+    render(json: AppSetting.all_enabled_for_user(user))
+  end
+
+  def show
+    user = User.find(app_setting_show_params[:user_id])
+    name =  app_setting_show_params[:name] 
+    render(json: {
+      name => AppSetting.enabled?(name: name, user: user)
+    })
+  end
+
+  private def app_setting_index_params
+    params.require(:user_id)
+  end
+
+  private def app_setting_show_params
+    params.permit(:user_id, :name)
+  end
```

Comment:
Should we be getting the user ID from `current_user` instead of expecting it to be passed in?  I can imagine plenty of front-end environments that don't actually know the details about the current user, and thus might not have a known ID to pass in.